### ISSUE TITLE
Incorrect FSF address in kernel driver files

### ISSUE THREAD

**Issue by z3ntu** [openrazer/openrazer#133]
```
[    5s] razer-kernel-modules-dkms.noarch: W: incorrect-fsf-address /usr/src/razer_chroma_driver-1.0.0/driver/razermouse_driver.c
[    5s] razer-kernel-modules-dkms.noarch: W: incorrect-fsf-address /usr/src/razer_chroma_driver-1.0.0/driver/razerfirefly_driver.c
[    5s] razer-kernel-modules-dkms.noarch: W: incorrect-fsf-address /usr/src/razer_chroma_driver-1.0.0/driver/razerkbd_driver.c
[    5s] razer-kernel-modules-dkms.noarch: W: incorrect-fsf-address /usr/src/razer_chroma_driver-1.0.0/driver/razerkraken_driver.c
[    5s] The Free Software Foundation address in this file seems to be outdated or
[    5s] misspelled.  Ask upstream to update the address, or if this is a license file,
[    5s] possibly the entire file with a new copy available from the FSF.
```
I'm sorry, Terry... (the copyright lines are convoluted as hell anyways)

**Comment by z3ntu** [comment]
> 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA

**Comment by z3ntu** [comment]
Fixed in e2820e8da75b053b4d078bc2976f728783c22894, thanks ^^
